store/store.go

```diff
@@ -69,5 +70,10 @@ func newStoreWithRetry(path string, maxRetries int) (kv.Storage, error) {
 		s, err = d.Open(path)
```

Comment:
How about adding some store details here? For example, the `path`.

Reply:
Done, PTAL.